workshop/azd-sample/src/ReleCloudLite.API/Data/ITicketContext.cs

```diff
@@ -0,0 +1,10 @@
+﻿using ReleCloudLite.Models;
+
+namespace ReleCloudLite.API.Data
+{
+    public interface ITicketContext
+    {
+        Task<IEnumerable<Ticket>?> GetTicketsAsync();
```

Comment:
nit: maybe use IAsyncEnumerable. Either way, returning a null IEnumerable should be replaced with an empty one